packages/renderer/src/lib/image/FilesystemLayerView.svelte

```diff
@@ -75,27 +77,31 @@ function modeString(data: ImageFile): string {
     (data.mode & 0o1000 ? 't' : data.mode & 0o001 ? 'x' : '-')
   );
 }
+let children = $derived(tree?.children);
+let file = $derived(tree?.data);
+let colorClass = $derived(getColor(tree));
+let arrowDown = $derived(expanded);
 </script>
 
 {#if layerMode || !tree.hidden}
   {#if root}
     {#if children}
       {#each children as [key, child] (key)}
-        <svelte:self root={false} margin={margin + 0.5} tree={child} layerMode={layerMode} />
+        <FilesystemLayerView root={false} margin={margin + 0.5} tree={child} layerMode={layerMode} />
```

Comment:
I confirm this is the right way (ref https://svelte.dev/docs/svelte/legacy-svelte-self)